.github/workflows/build.yml

```diff
@@ -12,32 +12,30 @@ jobs:
 
     runs-on: ${{ matrix.os }}
 
+    defaults:
+      run:
+        shell: bash -l {0}
+
     strategy:
       fail-fast: false
       matrix:
         os: [ubuntu-latest]
-        mpi: [nompi, openmpi]
+        mpi: [nompi, mpich]
```

Comment:
What's the reason for changing framework?

Reply:
Fetching `openmpi-mpifort` from conda-forge for whatever reason downgrades `gfortran` at the moment...

Reply:
> Can you just comment it up then?

You mean adding a comment in the build.yml file?

Reply:
Either the git log, or (and probably better) the yaml.

Reply:
Comment added to the build.yml file.